lib/internal/http2/core.js

```diff
@@ -1820,7 +1820,7 @@ class Http2Stream extends Duplex {
       req.handle = handle;
       const err = handle.shutdown(req);
       if (err === 1)  // synchronous finish
-        return afterShutdown.call(req, 0);
+        return afterShutdown.call(req);
```

Comment:
I haven't traced through the C++ side here, but Isuspect that in the async case, .oncomplete is called with an arg, a status. It happens that afterShutdown() doesn't use it ATM, but if it ever did, this sync call here would be wrong for not passing success/`0`. Maybe it would be better for afterShutdown() to add the arg, and then comment that its not used?

Reply:
@sam-github Your assumptions are correct :+1:

Reply:
Sure, I can make that change.